.github/workflows/docker_push.yml

```diff
@@ -0,0 +1,47 @@
+# from https://github.com/docker/build-push-action
+name: builddocker
+
+on:
+  push:
+    branches: master
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up python
+        uses: actions/setup-python@v2
+
+      # clone and patch anki library  
+      - name: Clone patch Anki
+        run: sh ./scripts/clone_patch_anki
+          
+      # set up and run pre-commit pre-commit run --all-files
+      - name: run pre-commit
+        uses: pre-commit/action@v2.0.3
+
+       # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+  
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ secrets.DOCKER_HUB_USERNAME }}/anki-sync-server-rs:latest
```

Comment:
Docker Hub doesn't support organisation-level credentials so you'll have to specify that you want to use the `ankicommunity` otherwise I'll push to my personal docker hub account.